stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -28,6 +28,13 @@ cFuncSuffix2 = {32: 'f', 64: 'd', 80: 'ld'}
 
 }%
 
+/// Returns `true` iff isspace(u) would return nonzero when the current
+/// locale is the C locale.
+@inlinable // FIXME(sil-serialize-all)
```

Comment:
If you add this as inlinable, then it has to be versioned. Since this is only used in one function, we could make it a nested function, but I'm not sure what the rules are there.

@jrose-apple, does the versioning rule also apply to a nested function inside an inlinable function?

Otherwise, you can just have it be a direct closure to the contains invocation.

Reply:
A nested function is always emitted into the client, like a closure.

Reply:
Including it directly within the closure seems like the simplest option to me, with the only real downside being that the file would be inconsistent.

Reply:
LGTM